platform/terraform/main.tf

```diff
@@ -0,0 +1,41 @@
+terraform {
```

Comment:
I struggle with trademarks in directory names. Must we use the name terraform? If we must, we must make sure the trademark policy permits it. I can take a look briefly. I just don't want to have to ever change the structure of the repository.

Reply:
also dangling. I moved to TF in the golden, golden-vm, and infrastructure directories. I'll clean this up.